frontend/src/static/css/style.scss

```diff
@@ -274,6 +294,18 @@ header {
     font-weight: bold;
     text-align: left;
 
+    &--index,
+    &--repo,
```

Comment:
If I don't separate out the index and the repo details then I can't provide colouring to just the index and the whole repo gets coloured.

Reply:
your `.border-index` already has `display: inline-block;` defined, hence the `display: inline-block;` can just be added to the other `&--repo {` definition below.

Reply:
Actually, when you put it like that I just realized, I don't need to create a universal style for this. Since we will just be using this style for group numbers. So, I will just create the style locally.